src/extension/app/aem-sidekick.js

```diff
@@ -41,6 +42,13 @@ export class AEMSidekick extends LitElement {
 
   async loadContext(config) {
     await this.appStore.loadContext(this, config);
+
+    const browser = detectBrowser(navigator.userAgent);
+    this.appStore.sampleRUM('click', {
+      source: 'sidekick',
+      target: `loaded:${browser}`,
```

Comment:
Dropped `platform` since we should already get this via RUM.

Reply:
What about the browser then? We should probably not try to detect this twice and differently.

Reply:
I didn't see anything in the RUM explorer detailing the browser. Could have missed it though. Maybe @trieloff can confirm if this is needed or not.

Reply:
We don't do browser detection as fine-grained as you do here